config/values/bursaries.yml

```diff
@@ -16,6 +16,7 @@ bursaries:
     languagesother: £26,000
     maths: £29,000
     music: £10,000
+    nonuk: £31,000
```

Comment:
would suggest putting this at the bottom of the list (as previously) as it is not really a subject!